src/app/auth/services/auth.service.spec.ts

```diff
@@ -56,6 +56,29 @@ describe("AuthService", () => {
     expect(req.request.method).toBe("GET");
   });
 
+  it("should should return true when no user is logged in", () => {
```

Comment:
This test description is a little confusing to me, "should return true when no user is logged in" shouldn't it return false instead? Also, remove the second should.

Reply:
thank you, I'm not it